src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -2268,6 +2268,9 @@ public void WritingTooLargeBase64Bytes(bool formatted, bool skipValidation)
 
             using (var jsonUtf8 = new Utf8JsonWriter(output, options))
             {
+                jsonUtf8.WriteStartArray();
```

Comment:
nit: can you instead move this to the `WriteBase64String` test instead? It makes more sense there.

That test writes an object, followed by 2 base-64 properties. Its fine to add a JSON array as well with 2 base-64 values.

This is similar to how we do it for other types like DateTime (in `WriteDateTimesValue`):
https://github.com/dotnet/corefx/blob/c38f2327e2d4e159acba06e23f38daed24661652/src/System.Text.Json/tests/Utf8JsonWriterTests.cs#L4927-L4933